packages/ui/src/domains/LandingPageBuilder/LandingPageBuilder.tsx

```
@@ -0,0 +1,2509 @@
+	font: string;
+	fontSize: number;
+	alignment: string;
+	bold: boolean;
+	italic: boolean;
+	underline: boolean;
+	textColor: string;
+	backgroundColor: string;
+	padding: string;
+	margin: string;
+	width: string;
+	onUpdate: (newProps: any) => void;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)